colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>Reading this, it's a bit unclear to me what's going on here. Do I always need to include both the <code>Logger</code>  and the <code>mixin</code>?  Is one more flexible than the other? Give a bit more detail here.</p><p> </p><p>Spread the code out, if you can, and interleave with text explaining what's going on, why you're doing it, and what else you could've done, if the situation were different.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>